test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -553,6 +529,64 @@ public async Task GivenListOfResources_WhenSearchedWithElementsAndSummarySetToFa
             ValidateBundle(bundle, patients);
         }
 
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenListOfResources_WhenSearchedSummarySetToData_ThenOnlySpecifiedPropertiesShouldBeReturned()
+        {
+            var tag = new Coding(string.Empty, Guid.NewGuid().ToString());
+
+            Patient patient = Samples.GetDefaultPatient().ToPoco<Patient>();
+            patient.Meta = new Meta();
+            patient.Meta.Tag.Add(tag);
+            await Client.CreateAsync(patient);
+
+            Bundle bundle = await Client.SearchAsync($"Patient?_tag={tag.Code}&{KnownQueryParameterNames.Summary}=data");
+            Assert.NotEmpty(bundle.Entry);
+            var returnedPatient = bundle.Entry[0].Resource as Patient;
+            Assert.Null(returnedPatient.Text);
+            Assert.Null(returnedPatient.Meta);
+            Assert.Null(returnedPatient.Id);
```

Comment:
This test doesn't pass for me. Meta and Id are included with summary=data.

Reply:
Yeah, I misread what 'data' does. It only removes "text".
In my defense, it's written in a somewhat vague way: 
[https://www.hl7.org/fhir/search.html#summary](https://www.hl7.org/fhir/search.html#summary)
> Remove the text element

they should put "" around text, same way as they put it in definition of text-summary.

---
In reply to: [587844939](https://github.com/microsoft/fhir-server/pull/1699#discussion_r587844939) [](ancestors = 587844939)